pkg/api/steve/supportconfigs/generator.go

```diff
@@ -54,12 +57,23 @@ func (h *GeneratorHandler) ServeHTTP(writer http.ResponseWriter, request *http.R
 		util.ReturnHTTPError(writer, request, http.StatusForbidden, "forbidden")
 		return
 	}
+	_, err = h.adapterUtil.GetRelease()
+	if err != nil {
+		if errors.Is(err, cspadapter.ErrNotFound) {
+			// if the adapter is not installed, return a 501 so the user knows to install the adapter
+			util.ReturnHTTPError(writer, request, http.StatusNotImplemented, "csp-adapter must be installed in order to to generate supportconfigs")
+			return
+		}
+		logrus.Warnf("[%s] Error when attempting to determine if adapter is installed, %s", logPrefix, err)
+		util.ReturnHTTPError(writer, request, http.StatusInternalServerError, "internal server error")
```

Comment:
Same here with the error text for common HTTP errors.

Reply:
Not sure if this comment was left in the right place, but I updated both L68 and L80 with this.

Good call, change made.